osu.Game.Rulesets.Osu/Objects/Drawables/DrawableSpinner.cs

```diff
@@ -49,15 +51,18 @@ public partial class DrawableSpinner : DrawableOsuHitObject
 
         private PausableSkinnableSound maxBonusSample;
 
+        private double classicMultiplier = 1.0;
```

Comment:
this can't be named "classic multiplier" if it respects the score display mode, call it `scoreDisplayModeMultiplier` or something